mnefun/_mnefun.py

```diff
@@ -249,6 +249,13 @@ class Params(Frozen):
     src_pos : float
         Default is 7 mm. Defines source grid spacing for volumetric source
         space.
+    on_missing : string
+        Can set to ‘error’ | ‘warning’ | ‘ignore’. Default is 'error'. Determine
+        what to do if one or several event ids are not found in the recording
+        during epoching. See mne.Epochs docstring for further details.
+    compute_rank : bool
+        Default is False. Set to True to compute rank of the noise covariance
+        matrix during inverse kernel computation.
```

Comment:
The only things that should go here are the arguments of `__init__`. Now would be a good time to add an `Attributes` section to this docstring that includes the things that are available but not in the constructor, starting with these two (can add others later). See:

https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
LMK if the change is what you are referring to. If so, should I just move all the arguments not part of `__init__` into the attributes section?

Reply:
> should I just move all the arguments not part of __init__ into the attributes section?

Yes if you have time please do